Fix CI file

matthieu526-scalingo opened the pull request.

matthieu526-scalingo commented:
> > I then tried to add the vendors but it wasn't the only issue. (BTW, why are they not commited)?
> 
> Vendors are not commited for library. You can have a look for example at [go-scalingo](https://github.com/Scalingo/go-scalingo/) or [go-utils](https://github.com/Scalingo/go-utils/).

Thanks! I removed them again.

EtienneM reviewed (approved):
@matthieu526-scalingo I fixed it. Mostly a problem with the golangci-lint version installed here being too old whereas the configuration file expects the most recent golangci-lint.

I updated our Dependabot configuration to take care of updating GitHub Actions.

The linter raised an issue that I fixed.

Is there anything needed here?

edit: I also updated the title to match our best practice

matthieu526-scalingo commented:
> @matthieu526-scalingo I fixed it. Mostly a problem with the golangci-lint version installed here being too old whereas the configuration file expects the most recent golangci-lint.
> 
> I updated our Dependabot configuration to take care of updating GitHub Actions.
> 
> The linter raised an issue that I fixed.
> 
> Is there anything needed here?
> 
> edit: I also updated the title to match our best practice

No nothing more to do here, thanks a lot! Once this is merged, I'll be able to create a new release.
